### PR TITLE
inference: update README.md

### PR DESCRIPTION
**Pull request by uMartinXu**
Signed-off-by: uMartinXu martin.xu@intel.com

### CODE REVIEW

**Comment by vbedida79** [comment]
+1

File: e2e/inference/README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+Intel AI inference e2e solution for OCP is built upon Intel® dGPU provisioning for OpenShift and Intel® Xeon® processors. The two following AI inference modes are used to test with the Intel Data Center GPU Card provisioning:
 * **Interactive Mode**
-[Open Data Hub (ODH)](https://github.com/opendatahub-io) and [Red Hat OpenShift Data Science (RHODS)](https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-data-science) provides Intel OpenVINO™ based [Jupiter Notebook](https://jupyter.org/) to help users interactively debug the inferencing applications or optimize the models with OCP using Intel Data Center GPU cards and Intel Xeon processors.
+[Open Data Hub (ODH)](https://github.com/opendatahub-io) and [Red Hat OpenShift Data Science (RHODS)](https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-data-science) provides Intel® OpenVINO™ based [Jupiter Notebook](https://jupyter.org/) to help users interactively debug the inferencing applications or optimize the models with OCP using Intel Data Center GPU cards and Intel Xeon processors.
```
Comment (vbedida79) [inline e2e/inference/README.md:6]: one typo Jupiter to Jupyter

Comment (uMartinXu) [reply e2e/inference/README.md:6]: Since it is only one typo, how about we merge this now, and send another PR to fix the typo?

**Comment by vbedida79** [comment]
+1

**Pull request merged**